Sources/MapboxCoreNavigation/NavigationLocationManager.swift

```diff
@@ -17,6 +17,8 @@ open class NavigationLocationManager: CLLocationManager {
             allowsBackgroundLocationUpdates = true
         }
     }
+    
+    public var simulatesLocation: Bool = false
```

Comment:
Remember to document this property. You can link it to the other property you’ve added (and vice versa).

```md
 - seealso: `NavigationMapView.simulatesLocation`
```